pkg/reporter/reporter.go

```diff
@@ -104,6 +104,7 @@ const (
 	UninstallPhaseStart                        CliStep = "uninstall.run.phase.start"
 	UninstallStepCheckRuntimeExists            CliStep = "uninstall.run.step.check-runtime-exists"
 	UninstallStepUninstallRepo                 CliStep = "uninstall.run.step.uninstall-repo"
+	UninstallStepRemoveGitIntegrations         CliStep = "uninstall.run.step.remove-git-integrations"
```

Comment:
@danielm-codefresh please review the addition of a new step to the uninstall flow.